tsl/test/shared/sql/dist_chunk.sql

```diff
@@ -0,0 +1,38 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+-- Test function _timescaledb_internal.create_chunk_replica_table
+
+-- A table for the first chunk will be created on the data node, where it is not present.
+SELECT chunk_name, data_nodes 
+FROM timescaledb_information.chunks 
+WHERE hypertable_name = 'dist_chunk_copy';
+
+-- Non-distributed chunk will be used to test an error
+SELECT chunk_name 
+FROM timescaledb_information.chunks 
+WHERE hypertable_name = 'conditions';
+
```

Comment:
What is the expected behavior for compressed chunks? Please add a test for compressed chunk.

Reply:
Compressed chunks are ignored. They are not even available on the access node.

Reply:
We should be able to copy/move compressed chunks. So, creation of a compressed chunk replica table on the DN needs to work, right?

Reply:
@nikkhils According to the decided plan copying compressed data will be next step after completing initial implementation of copy and move non-compressed chunk. It is more work to support compressed chunk, thus the work is done incrementally.

Reply:
Ok. Thanks for the explanation.

Reply:
I added a test with a compressed chunk, which demonstrates that it doesn't affect the result.